modules/src/joke.py

```diff
@@ -1,14 +1,30 @@
 import requests
 from templates.text import TextTemplate
+from random import choice
+import config
 
 def process(input, entities=None):
     output = {}
     try:
+        '''
         r = requests.get('http://tambal.azurewebsites.net/joke/random')
         data = r.json()
         output['input'] = input
         output['output'] = TextTemplate(data['joke']).get_message()
-        output['success'] = True
+        output['success'] = True        
+        '''
+        
+        with open(config.JOKES_SOURCE_FILE,"r") as file:
+            dump = file.read() # read the whole file
+            jokes_list = dump.split("\'''")[2].split("\n")
+            output['input'] = input
+            joke = choice(jokes_list).strip() #randomly choose a joke
+            if(len(joke)>2): #because there could be null values in the list
+                output['output'] = TextTemplate(joke).get_message()
+            else:
+                output['output'] = TextTemplate(choice(jokes_list).strip()).get_message() #choose again
```

Comment:
Actually its not likely gonna appear as that's hoe the choice model makes choices. 
Explaining further, the null values only appear when there is multiple \n character. (Different text editors issues)
The probability is quite little for a null value to appear. I've tested it myself before PR